source/develop/release-management/features/virt/dedicated-cpu.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Dedicated CPU Policies
+category: feature
+authors: tgolembi
+---
+
+# Dedicated CPU Policies
+
+## Summary
+
+This feature introduces policies for dynamically dedicating physical CPUs to
+the VM. While this can be (to some extent) done with static CPU pinning, the
+CPU pinning requires that the VM is pinned also to a particular host(s). It
+also ideally requires that no other VMs without pinning are running on the host
+to avoid sharing the pinned CPUs. Instead, this new feature allows the user to
+configure only the way how its physical CPUs should be assigned to the VM and oVirt
+will take care of dynamically allocating suitable physical CPUs as well as
+forbidding other VMs from running on dedicated CPUs.
+
+
+## Owner
+
+* Name: Tomáš Golembiovský
+* Email: tgolembi@redhat.com
+
+## Glossary
+
+  * *CPU pinning*: Feature in oVirt that allows you to statically pin vCPUs
+    to pCPUs by defining a pinning string in VM configuration (e.g.
+    `0#1_1#2-3`).
+  * *NUMA auto pinning policy*: Feature in oVirt that defines pinning policies
+    (*Pin* and *Resize and Pin*) that allow CPU pinning to automatically adapt
+    to the topology of pinned host. Just like CPU pinning, this is performed
+    once during VM configuration.
+  * *pCPU*: Physical CPU of a host.
+  * *vCPU*: Virtual CPU of a VM.
+
+
+## Description
+
+CPU pinning and NUMA auto pinning policies don't take into account current
+utilization of a host. Instead, it is a responsibility of the administrator to
+make sure that VMs running on a host are not pinned to same pCPUs and ideally
+also to make sure that no VMs without CPU pinning are running on the host. This
+makes the pinning process rather difficult and inflexible.
+
+To be able to have more flexibility in configuring how vCPUs of VMs use pCPUs
+this feature introduces several policies for dedicating CPUs. Compared to CPU
+pinning the advantages are that:
+
+  * VM does not need to be pinned to a specific host(s),
+  * takes into consideration existing VMs which solves one of the issues with
+    *Pin* NUMA auto pinning policy where several VMs can end up to be pinned
+    to same CPUs,
+  * it makes sure that VMs without any dedicated CPUs don't share CPUs with
+    VMs that have dedicated CPUs
+
+The available policies described below divide the set of available pCPUs into
+three sub-sets, There are the dedicated CPUs that are assigned only to a
+single VM. Then there are some unassigned CPUs that cannot be used by any
```

Comment:
"... only to a single VM." -- each of the pCPUs is assigned to  a single VM, not the whole pCPU group, right?